tests/unit/Tunnel.ts

```diff
@@ -48,7 +48,7 @@ registerSuite({
 		},
 
 		'stop a starting tunnnel'(this: Test) {
-			let timeout: NodeJS.Timer;
+			let timeout: number;
```

Comment:
I'm not sure this is valid. At least, now I get complaints about timeout not being of type NodeJS.Timer when building.

Reply:
Oh, wait, that could have been for something else.

Reply:
No, this seems to be a breaking change between TS 2.5.3 and TS 2.6. Drat.

Reply:
Apparently the trick to getting a predictable return type is to use `global.setTimeout` in Node and `window.setTimeout` in the browser.